gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Should be 1.1.0; this is a new feature

Reply:
> Should be 1.1.0; this is a new feature

Adjusted here: https://github.com/sonatype/sonatype-react-shared-components/pull/89/commits/3dfbe6a21e17758446d391f0cc5c0d1bfcb30ea5